gradoop-common/src/main/java/org/gradoop/common/util/NetworkHelper.java

```diff
@@ -70,7 +70,7 @@ public static String getLocalHost() {
         }
       }
     } catch (UnknownHostException | SocketException e) {
-      e.printStackTrace();
+      throw new IllegalStateException(e);
```

Comment:
Add a message to make it clear that an exception like this should not happen here.

Reply:
He took your comment literally :/

Reply:
This class is never used ... please remove it and it's test. @galpha Or do you know why this class exists?